examples_tests/13.BRDF_Explorer/createDerivMapFromBumpMap.cpp

```diff
@@ -0,0 +1,56 @@
+#include <irrlicht.h>
+#include "createComputeShader.h"
+#include "../source/Irrlicht/COpenGL2DTexture.h"
+#include "../source/Irrlicht/COpenGLDriver.h"
+#include "CComputeShaderManager.h"
+
+using namespace irr;
+
+video::IVirtualTexture* createDerivMapFromBumpMap(video::IVirtualTexture* _bumpMap, IrrlichtDevice* _device, float _heightFactor)
```

Comment:
shouldn't you factor this out to engine core in `video` namespace?

Reply:
@Crisspl ?

Reply:
resloved